server/graphql/v2/mutation/PayoutMethodMutations.ts

```diff
@@ -46,6 +46,7 @@ const payoutMethodMutations = {
           where: {
             data: { isManualBankTransfer: true },
             CollectiveId: collective.id,
+            isSaved: true,
```

Comment:
@kewitz I think you're the one who implemented this, what do you think about this?

TLDR ; should we fetch and update the removed bank transfer payout method (that's what we were doing) or ignore everything removed and create a new one (@SudharakaP's proposal)?

I feel Suds made a good call here but just want to make sure it won't break other things.

Reply:
Yes, I think this is a valid update.
Just keep in mind that every payout method created with `{ isManualBankTransfer: true }` is by default saved.

Reply:
@Betree @kewitz : Thanks for the comments. Are we good to merge this? 🤔

Reply:
I don't have the context of this anymore but I had no blocker at the time. Since @kewitz is good with this then I am too